src/components/DateRangePicker.js

```diff
@@ -79,7 +82,8 @@ class DateRangePicker extends React.Component {
     onClose () {},
     onDateSelect () {},
     placeholderText: 'Select A Date Range',
-    showDefaultRanges: false
+    showDefaultRanges: false,
+    theme: defaultTheme
```

Comment:
@bsbeeks can you remove this `defaultTheme` please? It's not necessary because of `StyleUtils.mergeTheme` (see https://github.com/mxenabled/mx-react-components/blob/7f2d4817c8121be95c1ce7540d324c90efd325fb/src/components/DateRangePicker.js#L222). Sorry to ask you to change yet another thing 😬

Reply:
I'll give it another go. But when no `theme` was provided in the props I got an error when the component tried to access `theme.BreakPoints`.